src/language/tekster_en.ts

```diff
@@ -841,6 +841,10 @@ export default {
   'banner.tittel.barnetilsyn':
     'Application for child care benefit for working single parents',
   'barnetilsyn.sidetittel': 'Application for child care benefit',
+  'barnetilsyn.søkerFraAugustTittel':
+    'Are you applying for childcare benefit from August ',
```

Comment:
```suggestion
    'Are you applying for childcare benefit from August [0]',
```

så slipper du å legge inn `variabel` i funksjonen ovenfor